### PR TITLE
Mafia: allow transitions between death states

### PR DESCRIPTION
**Pull request by ayiadvance**
Allow transitions between death states, as well as change the idea timer to 60 seconds, allow the lynch modifier of dead players to be changed, and start with reveals off.

### CODE REVIEW

File: server/chat-plugins/mafia.ts [hunk 1]
```diff
@@ -986,14 +984,17 @@ class Mafia extends Rooms.RoomGame {
 			player.destroy();
 			return;
 		}
-		this.dead[player.id] = player;
+		if (toEliminate in this.playerTable) this.dead[toEliminate] = this.playerTable[toEliminate];
```
Comment (AnnikaCodes) [inline server/chat-plugins/mafia.ts:987]: Should dead players be removed from the `playerTable`?

Comment (ayiadvance) [reply server/chat-plugins/mafia.ts:987]: They are, further down in the method

Comment (AnnikaCodes) [reply server/chat-plugins/mafia.ts:987]: Oh, okay.

**Review by AnnikaCodes — APPROVED** [review]
This looks fine to me.

**Pull request merged**